src/Illuminate/Http/Resources/Json/JsonResource.php

```diff
@@ -164,6 +164,16 @@ public function additional(array $data)
         return $this;
     }
 
+    /**
+     * Get the serialization options applied to the resource response.
+     *
+     * @return int
+     */
+    public function jsonOptions()
+    {
+        return 0;
```

Comment:
Returning 0 here means what exactly?

Reply:
It just means no options, which is the default for json serialization

Reply:
Why not null? It's pretty self-explanatory

Reply:
The result of this method will be passed to `json_encode` which expects an integer. Null is not possible here